### ISSUE TITLE
Issues with Magento 1.9.3.4

### ISSUE THREAD

**Issue by michaelwolz** [Shopify/shopify_transporter#111]
I tried to export customers from our old magento store. All permissions are set for the SOAP user, but I am getting a NoMethodError all the time. Is there anything I am missing? 

![screenshot 2018-11-02 11 54 35](https://user-images.githubusercontent.com/7479806/47911796-b6e93e80-de96-11e8-99ac-15ebe1b7a219.png)

**Comment by JordanVerasamy** [comment]
Looking into this. cc @michaelwolz

**Comment by JordanVerasamy** [comment]
Hey @michaelwolz -- I merged https://github.com/Shopify/shopify_transporter/pull/112. You should try again. You'll probably still get an error, but at least it'll be a useful error message this time. Let me know what happens, post a screenshot, and I can start trying to debug the root cause.

**Comment by michaelwolz** [comment]
Hey @JordanVerasamy, thank you :). I am getting the error "Extract error: Unable to obtain SOAP session ID from server."

![screenshot 2018-11-05 18 34 50](https://user-images.githubusercontent.com/7479806/48015682-fb403d00-e129-11e8-8b28-dab7e9dd58b0.png)

**Comment by JordanVerasamy** [comment]
By the way @michaelwolz, I'm deleting the screenshot you uploaded above since it has a session ID plainly visible. I'm like 98% confident that this isn't actually a security issue in any way. Still, in the future, please don't include credentials or session identifiers in issues you post here. Thanks!

**Comment by michaelwolz** [comment]
I can confirm this. Same error for me.

**Comment by surajreddy** [comment]
Thank you for trying this again so quickly @rahulr-codal and @michaelwolz !

I suspect that this is happening because the SOAP calls your instances support are different from the soap calls we're expecting.

Would it be possible for you to upload the WSDL for your Magento instances? This is already publicly available and usually does not contain any sensitive information. It lists the SOAP calls/methods your Magento instance supports.

You can get to the WSDL by going to `https://yourmagentourl.com/api/v2_soap?wsdl` in your browser, and then use your browser's *Save page* functionality to save a copy.

We can use this file to see where the calls from your instance are different from ours.

**Comment by michaelwolz** [comment]
Hello @surajreddy I attached the WSDL as txt file. Unfortunately GitHub is not supporting XML-Files in comments.

[v2_soap.txt](https://github.com/Shopify/shopify_transporter/files/2554563/v2_soap.txt)

**Comment by michaelwolz** [comment]
btw the error seems to be basically the same with products.

![screenshot 2018-11-07 11 40 47](https://user-images.githubusercontent.com/7479806/48126747-440b0980-e282-11e8-83da-5ae4238756b8.png)

**Comment by michaelwolz** [comment]
Any updates on this?

**Comment by JordanVerasamy** [comment]
Hey @michaelwolz -- some other internal Shopify stuff came up the last couple days and we had to deprioritize this temporarily. Support of this tool is on a best-effort basis and we can't really guarantee that we'll be able to address issues within users' timelines, especially since we can't directly reproduce the issue ourselves.

This is a developer tool that we open-sourced. We'll be helping where we can, but ultimately if you need faster support you should be willing to look at the code yourself, try to debug what's happening with your data (since you have access to your data and we don't), and actively try to debug the root cause of the issue.

**TL;DR: I'm hoping to be able to diagnose this later today, but waiting for us to fix every issue is not the right approach.**

That said: I took a look at the wsdl you posted. It seems drastically different than the wsdl of the Magento 1.9 instance that we're testing against. We're not sure why this is. I'm assuming you haven't (for example) modified your Magento install in any significant way?

**Comment by JordanVerasamy** [comment]
(Oh, also, I forgot to ask -- is this Magento Community or Enterprise?)

**Comment by JordanVerasamy** [comment]
Hey @michaelwolz I figured it out!

https://devdocs.magento.com/guides/m1x/api/soap/wsi_compliance.html

It looks like we've WS-I compliance mode set to "No", and you've got it set to "Yes".

Are you able to set this to "No" temporarily in order to run the Shopify Transporter extraction command? That'll be your best bet for a quick fix, although we're definitely going to add a flag to Transporter to allow users to specify whether their WS-I compliance mode is on or off eventually.

**Comment by rahulr-codal** [comment]
Hey @JordanVerasamy, 
It worked after setting WS-I compliance mode to "No" in Magento.

Thanks for the Help!

**Comment by surajreddy** [comment]
Marking this as closed as the WS-I compliance configuration seems to be the issue.
